suite-common/trading/src/thunks/exchange/sendTransactionThunk.ts

```diff
@@ -20,6 +20,7 @@ export type SendTransactionThunkProps = {
     trade: ExchangeTrade | undefined;
     decimals: number;
     shouldSendInSats: boolean | undefined;
+    isSLIP24Active?: boolean;
```

Comment:
```suggestion
    isSlip24Active?: boolean;
```
For better readability.

Reply:
Updated here https://github.com/trezor/trezor-suite/pull/19561/commits/a341ddeb6457422e8515fbd862bf13765efb8f78